installutils/helminstall/client.go

```diff
@@ -17,24 +17,25 @@ const (
 	TempChartFilePermissions = os.FileMode(0644)
 	TempChartPrefix          = "temp-helm-chart"
 	helmNamespaceEnvVar      = "HELM_NAMESPACE"
+	helmKubeContextEnvVar    = "HELM_KUBECONTEXT"
 )
 
 // This interface implements the Helm CLI actions. The implementation relies on the Helm 3 libraries.
 type HelmClient interface {
 	// Prepare an installation object that can then be .Run() with a chart object
-	NewInstall(namespace, releaseName string, dryRun bool) (HelmInstaller, *cli.EnvSettings, error)
+	NewInstall(kubeConfig, kubeContext, namespace, releaseName string, dryRun bool) (HelmInstaller, *cli.EnvSettings, error)
```

Comment:
I guess we could also stick these two new args on the constructor, and eliminate having to repeat them on every method invocation

Reply:
Let's leave it in the methods for now so that the same instance of HelmClient can be used to install across cluster/namespace, preventing the need for the caller to make a HelmClientFactory